core/src/main/java/discord4j/core/spec/MessageCreateSpec.java

```diff
@@ -158,13 +304,50 @@ public MessageCreateSpec setMessageReference(Snowflake messageId) {
     @Override
     public MultipartRequest asRequest() {
         MessageCreateRequest json = MessageCreateRequest.builder()
-                .content(content == null ? Possible.absent() : Possible.of(content))
-                .nonce(nonce == null ? Possible.absent() : Possible.of(nonce))
-                .tts(tts)
-                .embed(embed == null ? Possible.absent() : Possible.of(embed))
-                .allowedMentions(allowedMentionsData == null ? Possible.absent() : Possible.of(allowedMentionsData))
-                .messageReference(messageReferenceData == null ? Possible.absent() : Possible.of(messageReferenceData))
-                .build();
+            .content(contentBuilder.length() == 0 ? Possible.absent() : Possible.of(contentBuilder.toString()))
+            .nonce(nonce == null ? Possible.absent() : Possible.of(nonce))
+            .tts(tts)
+            .embed(embed == null ? Possible.absent() : Possible.of(embed))
+            .allowedMentions(allowedMentionsData == null ? Possible.absent() : Possible.of(allowedMentionsData))
+            .messageReference(messageReferenceData == null ? Possible.absent() : Possible.of(messageReferenceData))
+            .build();
         return new MultipartRequest(json, files == null ? Collections.emptyList() : files);
     }
+
+    @Nullable
+    private String getMarkdownTag(byte format) {
+        switch(format) {
```

Comment:
maybe even `MARKDOWN_BOLD_ITALIC` for `***` ?
![image](https://user-images.githubusercontent.com/12274096/104131336-6c20b680-537e-11eb-9d07-16839ddb1569.png)

Reply:
You can use appendFormatted("content", MessageCreateSpec.MARKDOWN_BOLD, MessageCreateSpec.MARKDOWN_ITALIC)

Reply:
Hmm, right... I'm guessing ordering will matter here, as if you specify a MARKDOWN_CODELINE first then your code will not be formatted. But that's neat nevertheless! 👍

Reply:
Both varargs and array lists are ordered, so if you put them in the correct order. It should be fine. Thanks for the reviews. 👍